tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+#          - Check if configuration file exists and contains expected values.
+#          - Check the message logs for "One time synchronization" occurance.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+
+    record_info("Check configuration", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
+    assert_script_run('ls /etc/systemd/system/yast-timesync.timer');
```

Comment:
as the `ls` does not provide any useful info you could use `file` or `stat`. i would go for the second. What do you think about it?

Reply:
Here the point was to just check the existence of the file. I think all three commands would serve this purpose the same.

Reply:
yes but you can provide some extra info in additional. i see it as two birds with one stone. feel free to resolve it if you disagree

Reply:
`file` will give us output: `ASCII text` and `stat` size, IO block, Access, modify change birth, but I don't see anything relevant for the purpose of this test unless something deeper want to be checked.